app/views/hyrax/my/_work_action_menu.html.erb

```diff
@@ -27,7 +27,7 @@
 
     <li role="menuitem" tabindex="-1">
       <%= display_trophy_link(current_user, document.id) do |text| %>
-        <i class='glyphicon glyphicon-star'></i> <%= text %>
+        <i class='glyphicon glyphicon-star'></i> <%= t("hyrax.dashboard.my.action.highlight") %>
```

Comment:
Hmm, in this case we throw away the value of `text`, which I *think* we care about? See here:

https://github.com/samvera/hyrax/blob/8cd726b1d16fa1b85eec02d2fbdaa2f71c31178a/app/helpers/hyrax/trophy_helper.rb#L9-L10

We can return one of two strings in this block. Maybe the translations should be added to the above helper instead of here, and then we can include a translation for the `"Unhighlight Work"` case?